internal/processor/otel/metadata.go

```diff
@@ -106,6 +106,10 @@ func translateResourceMetadata(resource pcommon.Resource, out *model.APMEvent) {
 		case "host.arch":
 			out.Host.Architecture = truncate(v.StringVal())
 
+		// session.*
+		case "session.id":
+			out.Session.ID = truncate(v.StringVal())
```

Comment:
This one will come from [span attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/README.md), not [resource attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/README.md).

In case you're not already familiar with the distinction:
 - resource attributes describe the entity producing observability data, and the environment in which that entity exists. This comprises things like `service.name`, `process.pid`, `os.name`, etc.
 - span attributes describe an individual operation, e.g. `http.method` for an HTTP request, or `db.statement` for a database query

Session IDs are event-specific, so we'll need to handle them in `traces.go`: I think both in `TranslateTransaction` and `TranslateSpan`.

Reply:
Oh, I see. Thank you for the explanation! 🙇 

I've updated the code!